routes/articles.js

```diff
@@ -218,4 +218,10 @@ articles.post(
   ArticleLikeController.dislike
 );
 
```

Comment:
#### Critical
- Please add a Swagger Doc for this route.

Reply:
done